app/controllers/anime_controller.rb

```diff
@@ -3,12 +3,14 @@
 class AnimeController < ApplicationController
   include MicrodataHelper
   def random
-    anime = Anime.where("age_rating <> 'R18+'").where(show_type: ["TV", "OVA", "ONA", "Movie"]).order("RANDOM()").limit(1)[0]
-    redirect_to anime
+    redirect_to Anime.sfw_filter(current_user)
+      .where(show_type: %w(TV OVA ONA Movie))
```

Comment:
Perhaps.  Either way it should probably be aligned (humans matter more than rubocop)